### ISSUE TITLE
investigate what actions for getting info from gihub missing

### ISSUE THREAD

**Issue by gorysko** [gorysko/gitlab_api_module#3]

**Comment by SergiySavarin** [comment]
we can add

**Comment by SergiySavarin** [comment]
We can add info about:
1. Events;
   - List public events
   - List repository events
   - List public events for an organization
   - List events that a user has received
   - List public events that a user has received
   - List events for an organization
2. Watchers:
   - List repo's watchers
   - List repositories being watched
3. List comments on a gist
4. Issues:
   - List issues for a repository
   - List assignees
   - List comments on an issue
   - List comments in a repository
   - List events for an issue
   - List events for a repository
   - List all labels for this repository
   - List labels on an issue
   - Get labels for every issue in a milestone
   - List milestones for a repository
5. Pull Requests:
   - List pull requests
   - List commits on a pull request
   - List pull requests files
   - List comments on a pull request
   - List comments in a repository
6. Repositories:
   - List user repositories
   - List contributors
   - List languages
   - List Teams
   - List Tags
   - List Branches
   - List collaborators
   - List commit comments for a repository
   - List commits on a repository
   - List deploy keys
   - List downloads for a repository
7. Users:
   - List followers of a user
   - List users followed by another user

**Comment by gorysko** [comment]
are you sure that all from your list are  not implemented yet?

**Comment by SergiySavarin** [comment]
Maybe list must be smaller but most of this item not included, like events, watchers, issues, pull requests, commit comments, users followers and etc. Maybe some of then we don't need at all... I recheck its again and update.

**Comment by gorysko** [comment]
ok  - all of this needed 
so - will You implement this actions?

**Comment by SergiySavarin** [comment]
yes I try to add it today, and commit changes, if have some questions call You
